test/no-hot/no-hot.test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const { run } = require('../utils/test-utils');
+const { stat, readFile } = require('fs');
+const { resolve } = require('path');
+describe('no-hot flag', () => {
+    it('shoul,d be successful when --no-hot is passed', (done) => {
+        const { stderr, stdout } = run(__dirname, ['--no-hot']);
+        expect(stderr).toBeFalsy();
+        expect(stdout).toBeTruthy();
+        expect(stdout).not.toContain('webpack/runtime/hot module replacement');
+
+        stat(resolve(__dirname, './bin/main.js'), (err, stats) => {
+            expect(err).toBe(null);
+            expect(stats.isFile()).toBe(true);
+            done();
+        });
+        readFile(resolve(__dirname, './bin/main.js'), 'utf-8', (err, data) => {
+            expect(err).toBe(null);
+            // check for absence of special functions invoked by HMR plugin only
+            expect(data).not.toContain('/* webpack/runtime/hot module replacement */');
+            expect(data).not.toContain('function hotApply');
+            expect(data).toContain('no-hot test');
```

Comment:
Let's avoid `function hotApply` and `no-hot test`. `webpack/runtime/hot module replacement` is enough

Reply:
@evilebottnawi updated :+1: